src/components/tops/TopbarSectionList.vue

```diff
@@ -39,6 +39,7 @@
             />
             <hand-coins-icon
               class="section-icon"
+              :stroke-width="1.5"
```

Comment:
Maybe add the `:stroke-width` to line the Line 17 too?
There is the samed icon component.